### PR TITLE
Implemented Signup / Login page for IdealClass

### PR DESCRIPTION
**Pull request by fasamany1**
This pull request adds a new signup/login page to the application.

### CODE REVIEW

File: src/components/login.jsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
+import React from "react";
+
+const Signup = () => {
```
Comment (driiisdev) [inline src/components/login.jsx]: this ought to be a login component?? isnt it? pls rectify this

**Review by driiisdev — CHANGES_REQUESTED** [review]
you have a duplicate of signup component on both login.jsx and signup.jsx file. pls rectify this error

File: src/components/login.jsx (hunk 1 above)
Comment (fasamany1) [reply src/components/login.jsx]: Yes, it is. Sorry for the inconvenience. I forgot to change the 'const' after copying and pasting from signup.

Comment (fasamany1) [reply src/components/login.jsx]: Please ignore the App.jsx file

Comment (fasamany1) [reply src/components/login.jsx]: It seems you've not reviewed the request. I can see it's pending.
Screenshot (110)

**Pull request merged**